README.md

```diff
@@ -104,6 +104,16 @@ with specfile.tags() as tags:
 # accessing tags in subpackages
 with specfile.tags('package devel') as tags:
     print(tags.requires)
+
+# working with changelog
+with specfile.changelog() as changelog:
+    # most recent changelog entry
+    print(changelog[0])
+    # making changes
+    changelog[1].date = datetime.date.today()
+    changelog[1].content.append('- another line')
+    # removing the oldest entry
+    del changelog[-1]
```

Comment:
I am wondering if using a reverse order wouldn't be more consistent/stable.

e.g.:
```python
>>> len(the_changelog)
3
>>> the_changelog[2]
v1.0 changelog entry
>>> the_changelog.add_changelog_entry()
>>> the _changelog[2]
v2.0 changelog entry
```

The old-to-new ordering will return the same changelog entries in this example. And using `[-1]` can still be nicely used to get the last one.

Reply:
Makes sense, I'll change it.